src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js

```diff
@@ -131,7 +131,7 @@ ${records
       .map(
         record =>
           `${
-            record.name === 'Adverse React/Allergy'
+            record.type === '11506-3'
```

Comment:
You should use constants here. We show the first set of fields if the `record.type` is one of:
- `loincCodes.PHYSICIAN_PROCEDURE_NOTE`
- `loincCodes.CONSULT_RESULT`